spec/integration/user_can_create_a_new_business_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.feature "Business can be created" do
+  scenario "by a registered user" do
+    user = create(:user)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).
+      and_return(user)
+
+    visit dashboard_path
+
+    expect(page).to have_link("Create New Business")
+    click_link "Create New Business"
+
+    fill_in "business[name]", with: "Cubby Stuffers"
+    fill_in "business[description]",
+             with: "A vertically integrated subscription snack service"
+
+    click_button "Create Business"
+
+    new_business = Business.last
+
+    expect(new_business.admins.include?(user)).to be true
+
+    visit business_path(new_business.slug)
+  end
```

Comment:
let's add an assertion that they see a pending message on the business dashboard when it has been created but not yet platform-admin approved